app/src/pages/inside/membersPage/modals/inviteUserModal/inviteUserModal.jsx

```diff
@@ -125,6 +148,18 @@ export class InviteUserModal extends Component {
               </FieldErrorHint>
             </FieldProvider>
           </ModalField>
+          {data.isProjectSelector && (
+            <ModalField label="Project" name="project" labelWidth={LABEL_WIDTH}>
+              <InputTagsSearch
+                minLength={1}
+                async
+                uri={this.projectSearchUrl}
+                value={this.state.projectFormattedValue}
```

Comment:
You don't need to store value in state as it comes from the redux-form. You can specify parse/format functions to convert value and pass them to `FieldProvider` as props. (Also, you need to wrap `InputTagsSearch` with `FieldProvider` to connect it to the form, see lines 141-149 for reference)